wagtail/admin/wagtail_hooks.py

```diff
@@ -874,6 +874,15 @@ def is_shown(self, request):
         )
 
 
+class PageTypesReportMenuItem(MenuItem):
+    def is_shown(self, request):
+        return getattr(
+            settings, "WAGTAIL_PAGE_TYPES_USAGE_REPORT_ENABLED", True
```

Comment:
Thought: Looking back on this again, It feels slightly strange that the setting should only impact the menu item visibility. The URL/view is still registered, so the report will still load if visited directly. If I disabled a report for a project, I think that might surprise me.

@jhonatan-lopes / @gasman How do you feel about dropping this setting? It's one thing less to test, and one thing less to document. I'm struggling to think of a project where the report wouldn't be valuable?

Reply:
Hey @ababic, it's fine for me. I was just following the template for the ageing pages report but if it doesn't make sense here, it should be fine to drop it

Reply:
Done!